cmd/create/cluster/cmd.go

```diff
@@ -941,6 +941,80 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+	if err != nil {
+		r.Reporter.Errorf("%s", err)
+		os.Exit(1)
+	}
+
+	billingAccounts := ocm.GenerateBillingAccountsList(cloudAccounts)
+
+	if billingAccount == "" && !interactive.Enabled() {
+		// if a billing account is not provided we will try to use the infrastructure account as default
+		if helper.ContainsPrefix(billingAccounts, awsCreator.AccountID) {
+			billingAccount = awsCreator.AccountID
+			r.Reporter.Infof("Using '%s' as billing account", billingAccount)
+			r.Reporter.Infof("To use a different billing account, add --billing-account xxxxxxxxxx to previous command")
+		}
+	}
+
+	if interactive.Enabled() {
+		if len(billingAccounts) > 0 {
+			billingAccount, err = interactive.GetOption(interactive.Input{
+				Question: "Billing Account",
+				Help:     cmd.Flags().Lookup("billing-account").Usage,
+				Default:  billingAccount,
+				Required: true,
+				Options:  billingAccounts,
+			})
+
+			if err != nil {
+				r.Reporter.Errorf("Expected a valid billing account: '%s'", err)
+				os.Exit(1)
+			}
+
+			billingAccount = ocm.RegexpReturnOnlyInteger(billingAccount)
+		}
+
+		if billingAccount == "" || !ocm.IsValidAWSAccount(billingAccount) {
+			r.Reporter.Errorf("Expected a valid billing account")
+			os.Exit(1)
+		}
+
+		// Get contract info
+		var contracts []*accountsv1.Contract
+		var contractEnableString string
+		for _, account := range cloudAccounts {
+			if account.CloudAccountID() == billingAccount {
+				contracts = account.Contracts()
+				for _, contract := range contracts {
+					for _, dimension := range contract.Dimensions() {
+						if ocm.GetDimensionValue(dimension, "four_vcpu_hour") > 0 ||
+							ocm.GetDimensionValue(dimension, "control_plane") > 0 {
+							contractEnableString = "   Contract enabled for this billing account. \n"
```

Comment:
maybe a typo here?  `contractEnabled`

Reply:
Hmm got it from the design. I'm thinking that as it's in a string, probably the two words are split.
![image](https://github.com/openshift/rosa/assets/26282541/4000fc1c-e9e2-45fc-9379-f0becba8b7c0)

Reply:
Changed the var.